xrpl-wasm-std/src/core/submit/authorize.rs

```diff
@@ -0,0 +1,39 @@
+// pub fn authorize(recipient: &AccountID, amount: &Amount) -> i32 {
```

Comment:
Use it or lose it.

This and transfer are here because while you can extend Amount to transfer it doesnt transfer NFTs so should the functions be generics that take amounts as a variable rather than extending the `Amount` class.